service/api/wp-content/mu-plugins/advanced-post-cache.php

```diff
@@ -243,4 +243,4 @@ function dont_clear_advanced_post_cache() {
 // Don't clear Advanced Post Cache for a new comment - temp core hack
 // http://core.trac.wordpress.org/ticket/15565
 add_action( 'wp_updating_comment_count', 'dont_clear_advanced_post_cache' );
-add_action( 'wp_update_comment_count'  , 'do_clear_advanced_post_cache'   );
\ No newline at end of file
+add_action( 'wp_update_comment_count'  , 'do_clear_advanced_post_cache'   );
```

Comment:
@jeffpaul I would not have merged this, the whole file is a copy and paste from an existing plugin. Making changes to it is unnecessary. Plus this style change is not inline with our coding standards. We don't align actions like this.